odoo/_monkeypatches/__init__.py

```diff
@@ -0,0 +1,34 @@
+# ruff: noqa: F401, PLC0415
+# ignore import not at top of the file
+import os
+import time
+from .evented import patch_evented
```

Comment:
patch_evented must be called be calling `import os` as gevent patches it: https://www.gevent.org/api/gevent.monkey.html#gevent.monkey.patch_all, https://www.gevent.org/api/gevent.monkey.html

Reply:
`gevent.monkey.patch_all()` already imports libraries it will patch and os=True by default.